docs/communications/examples/isr-search.rst

```diff
@@ -2,8 +2,7 @@
 ISR Search
 ==========
 
-.. note::
-  If you have not read the information provided for all of these examples, `please do so <../examples.html>`_ before proceeding on reading this example.
+.. note:: If you have not read the information provided for all of these examples, `please do so <../examples.html>`__ before proceeding on reading this example.
```

Comment:
Made paragraph-level `note` markup inline. This highlights the text differently when viewed in the editor.